hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: minor
+
+This release adds support for example databases backed by Github Actions artifacts. These can be used by developers to access the same example databases being used by a CI workflow, in a read-only manner.
```

Comment:
Let's make sure there's a cross-reference, so that interested readers can jump to the docs:

```rst
This release adds :class:`GitHubArtifactDatabase`, which allows you to ...

Thanks to Agustín Covarrubias for this feature!
```